jobs/sync-anime/sync-anime.go

```diff
@@ -58,7 +58,13 @@ func sync(data *kitsu.Anime) *arn.Anime {
 	anime.EndDate = attr.EndDate
 	anime.EpisodeCount = attr.EpisodeCount
 	anime.EpisodeLength = attr.EpisodeLength
-	anime.Status = attr.Status
+
+	if strings.EqualFold(attr.Status, "tba")  || strings.EqualFold(attr.Status, "unreleased") {
```

Comment:
A simple string comparison would do, attr.Status == "tba"

Reply:
is "tba" and "unreleased" all lower case or does it start with a capital?

Reply:
Kitsu status is all lowercase. We use the same lowercased status.